src/Payment/Service.php

```diff
@@ -15,19 +16,26 @@ interface Service
      * @param Token $token
      * @param int $amount
      * @param string $currency
-     * @param string|null $description
      * @param array $metadata
+     * @param Token|null $customer
+     * @param string|null $description
      *
      * @return Payment
      */
     public function create(
         Token $token,
         int $amount,
         string $currency = 'GBP',
-        string $description = null,
-        array $metadata = []
+        array $metadata = [],
+        $customer = null,
```

Comment:
can be typehinted

Reply:
Forgot to update that after the Token class went abstract 👍